database/seeds/JaminanTableSeeder.php

```diff
@@ -11,6 +11,6 @@ class JaminanTableSeeder extends Seeder
      */
     public function run()
     {
-        factory(Simpin\Infrastructure\Repository\Jaminan\JaminanEloquentRepository::class,10)->create();
+        // factory(Simpin\Infrastructure\Repository\Jaminan\JaminanEloquentRepository::class,10)->create();
```

Comment:
I think we just have to delete this seeder because it is unused.

Reply:
dah cok https://github.com/AxlAdilla/Simpin/pull/1/commits/3673a529627b124e136fa1b13ecf667e35b8d004